crates/sui-core/src/authority.rs

```diff
@@ -467,7 +468,10 @@ impl AuthorityState {
         // for every tx.
         let tx_guard = self.database.acquire_tx_guard(&certificate).await?;
 
-        self.process_certificate(tx_guard, &certificate).await
+        log_if_err!(
+            self.process_certificate(tx_guard, &certificate).await,
+            ?digest
```

Comment:
@bmwill this macro will not accept an optional `,` after the last arg - do you know how to fix that?

Reply:
As I sent in slack I'd actually recommend not writing a new macro for this and instead use [`tap`](https://docs.rs/tap/latest/tap/index.html)